addons/crm_partner_assign/wizard/crm_forward_to_partner.py

```diff
@@ -1,16 +1,41 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp.osv import fields, osv
-from openerp.tools.translate import _
+from openerp import api, fields, models, _
 from openerp.exceptions import UserError
 
 
-class crm_lead_forward_to_partner(osv.TransientModel):
+class CrmLeadForwardToPartner(models.TransientModel):
     """ Forward info history to partners. """
     _name = 'crm.lead.forward.to.partner'
 
-    def _convert_to_assignation_line(self, cr, uid, lead, partner, context=None):
+    @api.model
+    def default_get(self, fields):
+        CrmLead = self.env['crm.lead']
+        MailTemplate = self.env['mail.template']
+        try:
+            template_id = self.env.ref('crm_partner_assign.email_template_lead_forward_mail')
+        except ValueError:
```

Comment:
no more try and exception. if you want to raise exception  you can just pass 'raise_if_not_found=True' in self.env.ref as second argument.